ietf/static/ietf/js/edit-meeting-schedule.js

```diff
@@ -1,5 +1,13 @@
+/* globals alert, jQuery, moment */
 jQuery(document).ready(function () {
     let content = jQuery(".edit-meeting-schedule");
+    /* Drag data stored via the drag event dataTransfer interface is only accessible on
+     * dragstart and dragend events. Other drag events can see only the MIME types with
+     * data. Use a non-registered type to identify our session drags. Unregistered MIME
+     * types are strongly discouraged by RFC6838, but we are not actually attempting to
+     * exchange data with anything outside this script so that really does not apply. */
+    const dnd_mime_type = 'text/x.session-drag';
```

Comment:
+1 awesome points for using the MIME time instead of text processing on the value of the data as a signal that this is the right type of drag event.